.github/workflows/docs-production.yml

```diff
@@ -76,11 +76,17 @@ jobs:
           mkdir -p docs/docs
           ./build_current_version.sh
 
-      - name: Publish preview to Netlify
-        run: |
-          cd netlify
-          npm install netlify-cli -g
-          netlify deploy --prod --build --context production
+      - name: Publish production to Netlify
+        uses: nick-fields/retry@v3
+        with:
+          timeout_seconds: 1600
+          max_attempts: 3
+          retry_on: error
+          command: |
+            cd docs
+            cd netlify
+            npm install netlify-cli -g
+            netlify deploy --prod --build --context production
```

Comment:
Won't this fail by trying to change the directory to `docs/netlify` while being in this directory after the first try?

Reply:
Seems like it's a separete context each retry, so my guess is it won't . Be to be honest I'm not 100% sure. There is no native approach for retry in gh actions, so it needs to be using this. As far it is working in the staging environment

Reply:
Wouldn't it make more sense to move the `cd`s and `npm install`s to the previous step to be sure? The thing that's failing is not the directory changes nor the npm installation, right?